database/models/files.db.php

```diff
@@ -28,7 +28,7 @@ class SharedFile
     public Permission $permission_level = Permission::USER;
 
     #[ManyToOne]
-    public Race|null $race = null;
```

Comment:
Here we could actually just change that since we are not supposed to use this foreign key yet, but create a migration file for this.

Reply:
I would actually suggest dropping this change. We will add a migration in a later PR: This ensures that we dont mess up the DB too much